bigtop-packages/src/rpm/qfs/SPECS/qfs.spec

```diff
@@ -0,0 +1,360 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+%define qfs_name qfs
+
+%define bin_dir /usr/bin
+%define lib_dir /usr/lib/qfs
+%define etc_dir %{_sysconfdir}/qfs
+%define include_dir /usr/include
+%define data_dir /usr/share/qfs
+%define var_dir /var
+%define qfs_services chunkserver metaserver webui
+
+%global HADOOP_VERSION %{hadoop_version}
+
+%if %{?!HADOOP_VERSION:1}0
```

Comment:
We do pass it through from the bigtop bom. See my changes to packages.gradle to see how this is handled and flows through.